spec/mailboxes/comments_mailbox_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe CommentsMailbox, type: :mailbox do
+  subject { process(mail) }
+
+  let(:ticket) { create(:ticket) }
+  let(:support_email) { "ticket-#{ticket.id}@#{ApplicationMailer::SUPPORT_DOMAIN}" }
+  let(:email_subject) { ticket.subject }
+  let(:body) { Faker::Lorem.paragraphs(number: 2).join(',') }
+  let(:mail) do
+    Mail.new(from: user.email,
+             to: support_email,
+             subject: email_subject,
+             body:)
+  end
+  let(:user) { ticket.user }
+
+  context 'with valid params' do
+    it 'marks email as delivered when the from receipt is monterrey-(#ticket)' do
```

Comment:
Cristian asked the same in the previous PR and I did not find an answer to his question. Why does it reference "monterrey" in several places? What does it mean?

Reply:
I assume an account with the name of my condominium. Fixed